docs/en/04_Changelogs/4.7.0.md

```diff
@@ -0,0 +1,48 @@
+# 4.7.0 (Unreleased)
+
+## Overview
+
+- [Experimental support for PHP 8](#experimental-support-for-php-8)
+- [Support for Symfony 4 Components](#support-for-symfony-4-components)
+- [Default MySQL collation updated](#default-mysql-collation-updated)
+
+## New features
+
+### Experimental support for PHP 8
+
+You can now run Silverstripe CMS on PHP 8, which is due for release in November 2020. PHP 8 includes
+many exciting new features like named parameters, attributes, and union types.
+
+Upgrading to PHP 8 on projects using `SapphireTest` for unit testing requires adoption of the
+[`sminnee/phpunit`](https://packagist.org/packages/sminnee/phpunit) fork, and Prophecy is not
+currently supported. A future release will adopt the current mainline version of PHPUnit.
+
+Note that this support does not yet extend to non-core modules, and that PHP 8 is not yet stable, so
+compatibility issues could arise in future PHP builds. Additionally, support for the new JIT
+functionality has not yet been explored. If you find any bugs when running CMS Recipe 4.7.0 against
+PHP 8, please raise an issue on GitHub.
+
+### Support for Symfony 4 Components
+
+Symfony 3 will become unsupported early next year, so Silverstripe CMS is now forwards-compatible
+with Symfony 4 components. This is a largely transparent upgrade, but you may encounter errors
+related to strings starting with `%` in YML files, which can be easily rectified by wrapping all
+affected strings in quotes.
```

Comment:
If you wish to keep using the old Yaml parser, run "composer require symfony/yaml:^3" on your project to prevent installation of version 4.